src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_map.c

```diff
@@ -317,7 +317,7 @@ void KaleidoScope_UpdateDungeonCursor(PlayState* play) {
             pauseCtx->cursorColorSet = PAUSE_CURSOR_COLOR_SET_WHITE;
             oldCursorPoint = pauseCtx->cursorPoint[PAUSE_MAP];
             if (pauseCtx->stickAdjX > 30) {
-                pauseCtx->unk_298 = 4.0f;
+                pauseCtx->cursorShrinkRate = 4.0f;
                 if (pauseCtx->cursorSpecialPos == PAUSE_CURSOR_PAGE_LEFT) {
                     func_80821A04(play);
```

Comment:
Hmm seems to get called when moving pages from the dungeon map page? Perhaps we can name it based on that.

Reply:
Yeah, it getting called when switching the page makes sense. It just updates a bunch of data/misc variables when the page gets switched, but then there's also `KaleidoScope_SwitchPage`? So I'm not sure. Should definitely have `SwitchPage` in its name somehow

Reply:
So looking at it I would suggest `KaleidoScope_MoveCursorFromSpecialPos`. (It actually doesn't seem to be about switching pages, just moving from the page turn buttons (R and Z)).